libs/designer/src/lib/core/actions/bjsworkflow/serializer.ts

```diff
@@ -455,6 +455,14 @@ interface FunctionConnectionInfo {
   };
 }
 
+interface OpenApiConnectionInfo {
+  host: {
+    apiId: string;
+    connectionName: string;
```

Comment:
should this be `connection` if it's related to this? https://github.com/Azure/LogicAppsUX/blob/main/types/LogicAppsV2.d.ts#L108

Reply:
Interesting find. This makes sense. I can actually just use that type.